.travis.yml

```diff
@@ -1,24 +1,26 @@
+os: linux
+dist: xenial
 language: php
 
 php:
-#  - 5.3
-#  - 5.4
-#  - 5.5
   - 5.6
   - 7.0
   - 7.1
   - 7.2
   - 7.3
   - 7.4
   - hhvm
```

Comment:
HHVM has no PHP support anymore and could therefore be ignored IMHO (source from 2018: https://hhvm.com/blog/2018/09/12/end-of-php-support-future-of-hack.html)

Reply:
Good idea too, I'll drop it and add nightly instead